### PR TITLE
Leaderboard: hit the API

### PR DESCRIPTION
**Pull request by zhorabay**
- [ ] In this branch, I:
- Created a new game with the name 'Monopoly' by using the API.
- Saved the ID of the game that will be returned by API.
- Implemented the "Refresh" button that will get all scores for a game created by me from the API (receiving data from the API and parsing the JSON).
- Implemented the form "Submit" button that will save a score for a game created by me (sending data to the API).
- Used arrow functions instead of the function keyword.
- Used async and await JavaScript features to consume the API.

### CODE REVIEW

File: src/modules/apiRequest.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    body: JSON.stringify({
+      username: playerName,
+      score: playerScores,
+    }),
+    headers: {
+      'Content-Type': 'application/json; charset=UTF-8',
+    },
+  });
+};
+
```
Comment (DammyShittu) [inline src/modules/apiRequest.js]: - [ ] Your app does not work as required. When the app is run, a new game is not created and when a score is added it is not added to the API either. Kindly fix this.

- [ ] When I click on the refresh button nothing is gotten from the API either.

  image

**Review by DammyShittu — CHANGES_REQUESTED** [review]
Hi @zhorabay,

## Highlights
✔️ Async and await for functions

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
- [ ] While your header links work as required you repeated them. Kindly fix this.

  image

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/apiRequest.js (hunk 1 above)
Comment (zhorabay) [reply src/modules/apiRequest.js]: @DammyShittu I submitted the work to ask help, can you help me to fix this? I think I did everything correct but it shows error 400. Can you tell me what exactly is my mistake? what should i change? I know I can't add the score but why?

**Review by Gambit142 — APPROVED** [review]
Hi @zhorabay 👋,

## Status: Approved 🟢   

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

### Highlights
- Setup linters correctly ✔️ 
- Application can post data to endpoint ✔️ 
- Application can get data from endpoint ✔️ 

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### Please tag me in your question so I can receive the notification. You can tag me by typing @Gambit142

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**